apps/website/src/composables/iopay.ts

```diff
@@ -0,0 +1,58 @@
+import Antenna from 'iotex-antenna'
+import { WsSignerPlugin } from 'iotex-antenna/lib/plugin/ws'
+import { toRau } from 'iotex-antenna/lib/account/utils'
+import useWallet from '@/composables/wallet'
+
+// TODO: Figure out why these refe aren't updating
+// const { toAddress, amount } = useWallet()
+
+export default function useIopay() {
+  const signer = new WsSignerPlugin()
+  const antenna = new Antenna('http://api.testnet.iotex.one:80', {
+    signer,
+  })
+
+  const getIoPayAccounts = async () => {
+    return await signer.getAccounts()
+  }
+
+  const sendIoPayTransaction = async (toAddress: string, amount: string) => {
+    try {
+      const transResp = await antenna?.iotx.sendTransfer({
+        to: `${toAddress}`,
+        from: antenna.iotx.accounts[0].address,
+        value: toRau(amount, 'Iotx'),
+        gasLimit: '100000',
+        gasPrice: toRau('1', 'Qev'),
+        // chainId: 'IoTeXChain',
```

Comment:
Try `chainId: 2` for testnet. I don't think their code uses this, but the type requires it.